compiler/lib/src/main/scala/ast/Ast.scala

```diff
@@ -587,7 +611,63 @@ object Ast {
 
   }
 
-  /** Port matching specifier */
+  /** Initial state Specifier */
+  final case class SpecInitial(
+    action: Option[Ident],
+    state: Ident
```

Comment:
State needs to be a qualified identifier, not a simple identifier. Also, in the case of an initial specifier, we revised the spec so that the syntax is `initial [enter-expression]`.